.github/workflows/maven.yml

```diff
@@ -65,7 +57,7 @@ jobs:
         run: sudo apt install -y --no-install-recommends graphviz
 
       - name: Build with Maven
-        run: mvn install -Dmaven.test.skip=true -B -V -fae -q
+        run: mvn install -Dmaven.test.skip=true -B -V -fae -q -Djava.awt.headless=true
```

Comment:
Without setting the `headless` mode on command line the combination of java and maven 3.9.7 would fail with `NCDFE: Could not initialize class java.awt.GraphicsEnvironment$LocalGE` during the `*DatabaseTest`s. It seems that all over places where `headless` mode is configured are ignored e.g. `maven.config`, `jvm.config` or `MAVEN_OPTS`.

Reply:
this might be because the test instantiates a new Maven process per test (not sure, it's been a long time)